### DISCUSSION TITLE
You need to install Docker Desktop first.

### DISCUSSION THREAD

**Discussion by AnasElGouraini** [ichi4096/vivado-on-silicon-mac#67] [Q&A]
Hello, I can't run vivado because the program tells me that I haven't installed docker. However, I haven't touched anything. Then I uninstalled and reinstalled it but I still get the same error message ‘You need to install Docker Desktop first.’

**Comment by ejortega** [comment]
@AnasElGouraini have you installed [Docker Desktop](https://www.docker.com/products/docker-desktop/) yet?
